splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/IndexPrefixIterationIT.java

```diff
@@ -302,5 +302,34 @@ public void testParameterized() throws Exception {
 
     }
 
-
+    @Test
+    public void testIndexLookupNoNPE() throws Exception {
```

Comment:
Does this test case test both start key and stop key cases? I mean, would this test pass if we don't fix the stop key part?

Reply:
@ascend1 Yes.  With the stop key portion of the fix commented out, the test will fail, so both portions are needed.